src/main/java/io/github/fabriccommunity/everything/annotations/Nullable.java

```diff
@@ -0,0 +1,17 @@
+package io.github.fabriccommunity.everything.annotations;
+
+import static java.lang.annotation.ElementType.*;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Target maybe or return null
+ */
+@Target({FIELD,METHOD,PARAMETER})
+@Retention(RetentionPolicy.CLASS)
+public @interface Nullable {
```

Comment:
Ah yes so we have 2 null annos now

Reply:
it's good to give developers options to pick from

Reply:
entirely agreed, will merge in the next 30 minutes if nobody complains
the race is on!

Reply:
nice